src/devtools/packages/devtools-reps/object-inspector/utils/index.ts

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+import { ValueItem } from "./value";
+import { KeyValueItem } from "./keyValue";
+import { MODE } from "../../reps/constants";
+import { LoadingItem } from "./loading";
+import { ContainerItem } from "./container";
+import { Pause } from "protocol/thread";
+import { ObjectInspectorItemProps } from "../components/ObjectInspectorItem";
+
+const {
+  REPS: { Rep, Grip },
+} = require("../../reps/rep");
+
+export interface LabelAndValue {
+  label?: React.ReactNode;
+  value?: React.ReactNode;
+}
+
+export interface IItem {
+  readonly type: string;
+  name: string | undefined;
```

Comment:
This is on purpose: I wanted to express that every `IItem` implementation should have a `name` property, even if it's `undefined`. Otherwise typescript would complain about `item.name` as soon as you add an item type that does not have a `name` property.